src/ccl_neutrinos.c

```diff
@@ -170,14 +170,7 @@ double ccl_Omeganuh2 (double a, int N_nu_mass, double* mnu, double T_CMB, int* s
 
   Tnu = T_CMB*pow(4./11.,1./3.);
   a4 = a*a*a*a;
-  // Check if mnu=0. We assume that in the massless case mnu is a
-  // pointer to a single element and that element is 0. This should in principle never be called.
-  if (mnu[0] < 0.00017) {  // Limit taken from Lesgourges et al. 2012
```

Comment:
But did you actually have to remove this line? It does seem to be sensible to revert to the default calculation ofr low enough mass.

Reply:
Fair point. I was being a bit lazy by removing it as it almost never should get hit (only for the case where we have to post-hoc fudge equal neutrino masses for cosmic emu, I think). But I've reintroduced that case now (in the below loop so we make sure to check them all and not just the first as was causing some of the original weird behaviour).

Reply:
Thoughts @slosar? If you're happy could you approve?